bittensor/cli.py

```diff
@@ -92,6 +92,7 @@
             "regen_coldkeypub": RegenColdkeypubCommand,
             "regen_hotkey": RegenHotkeyCommand,
             "faucet": RunFaucetCommand,
+            "update_wallet": UpdateWalletCommand,
```

Comment:
Do we need to NaCL to the requirements?

Reply:
One followup, what are the python requirements for NaCl? If ansible vault is being depreciated, we might be able to bring back python 3.8.

Reply:
cannot deprecate ansible vault, cause we still need it to decrpyt some old wallet to update the wallet, 
other then that, we may need to keep supporting the old wallets